helm/prometheus-rules/templates/alerting-rules/dex.rules.yml

```
@@ -22,3 +22,27 @@ spec:
         severity: page
         team: rainbow
         topic: dex
+    - alert: DexSecretExpired
+      annotations:
+        description: '{{`Dex operator running on {{ $labels.installation }} failed to renew secret of {{ $labels.app_registration_name }} for {{ $labels.app_owner }} on provider {{ $labels.provider_type }}.`}}'
```

I think we should use the same name for the component everywhere.

```suggestion
        description: '{{`dex-operator running on {{ $labels.installation }} failed to renew secret of {{ $labels.app_registration_name }} for {{ $labels.app_owner }} on provider {{ $labels.provider_type }}.`}}'
```